api/pkg/apis/v1alpha1/managers/solution/solution-manager.go

```diff
@@ -324,6 +324,9 @@ func (s *SolutionManager) Reconcile(ctx context.Context, deployment model.Deploy
 
 	s.saveSummaryProgress(ctx, deployment, summary, namespace)
 	defer func() {
+		if deployment.IsDryRun {
+			summary.SuccessCount = 0
```

Comment:
I think the default successCount is 0, no?

Reply:
This is set successCount back to 0 after all the deployment plans are finished. Why we are doing this is to show "deployed" field in status always as 0 for dryrun case